lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -43,7 +43,7 @@ class TargetIntegrator
 
         # @return [Integer] the maximum number of input and output paths to use for a script phase
         #
-        MAX_INPUT_OUTPUT_PATHS = 1000
+        MAX_INPUT_OUTPUT_PATHS = 10000
```

Comment:
I don't see much of a problem with this but why not use Xcode 10 and `xcfilelists` which dont have this problem?